depthai_datatype_msgs/msg/RawCameraControl.msg

```diff
@@ -0,0 +1,20 @@
+uint64 cmdMask
+uint8 autoFocusMode
+uint8 lensPosition
+ManualExposureParams expManual
+RegionParams aeRegion
+RegionParams afRegion
+uint8 awbMode
+uint8 sceneMode
+uint8 antiBandingMode
+uint8 effectMode
+bool aeLockMode
+bool awbLockMode
+int8 expCompensation
+int8 brightness
+int8 contrast
+int8 saturation
+uint8 sharpness
+uint8 lumaDenoise
+uint8 chromaDenoise
+uint8[] data
```

Comment:
Is there any impact of the order? If not, can we arrange them in any order to minimize total message size? Eg:
```
bool
int32
# 4 + 4 byte long (bool takes entire 4 bytes, 3 bytes of padding)
```
Contrast with
```
int32
bool
# 4 + 1 byte long (bool takes just 1 byte)
```

Reply:
publisher didn't care about the order, 
I am not sure about subscriber.

If the order is important in subscriber, we need to make it in alphabetical order. Otherwise, I prefer the field to be in the same order as the definition in depthai headers to padding for memory efficiency.